command/perf_send.go

```diff
@@ -0,0 +1,104 @@
+package command
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/rest/client"
+	"github.com/evergreen-ci/poplar"
+	"github.com/evergreen-ci/poplar/rpc"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+)
+
+type perfSend struct {
+	// AwsKey and AwsSecret are the user's credentials for authenticating
+	// interactions with s3. These are required if any of the tests have
+	// artifacts.
+	AWSKey    string `mapstructure:"aws_key" plugin:"expand"`
+	AWSSecret string `mapstructure:"aws_secret" plugin:"expand"`
+
+	// Region is the s3 region where the global bucket is located. It
+	// defaults to "us-east-1".
```

Comment:
Where does it actually set the default region? Is it within Poplar/Cedar?

Reply:
Within poplar.